Client/multiplayer_sa/CMultiplayerSA_WidescreenFix.cpp

```diff
@@ -0,0 +1,1111 @@
+    // Misc
+    int m_dwMiscWidth[] = {
+        0x5733FD,            // 0 StretchX
+        0x574761,            // 1 Radio Icons
+        0x5747A6,            // 2 Radio Icons
+        0x574857,            // 3 Radio Icons
+        0x5748AA,            // 4 Radio Icons
+        0x5765C0,            // 5 2d Brief triangles
+        0x576603,            // 6 2d Brief triangles
+        0x576646,            // 7 2d Brief triangles
+        0x576689,            // 8 2d Brief triangles
+        0x57672F,            // 9 2d Brief triangles
+        0x576772,            // 10 2d Brief triangles
+        0x5767B5,            // 11 2d Brief triangles
+        0x5767F8,            // 12 2d Brief triangles
+        0x574ECC,            // 13 Stats bars
+        0x574F30,            // 14 Stats bars
+        0x719D2D,            // 15 Font fixes
+        0x719D94,            // 16 Font fixes
+        0x719DD1,            // 17 Font fixes
+        0x719E0E,            // 18 Font fixes
+        0x719E4B,            // 19 Font fixes
+        0x719E6F,            // 20 Font fixes
+        0x719E97,            // 21 Font fixes
+        0x719C0D,            // 22 Font fixes
+        0x719C6E,            // 23 Font fixes
+        0x7288F5,            // 24 BarChart fixes
+        0x728941,            // 25 BarChart fixes
+        0x573F93,            // 26 DrawWindow header
+        0x573FF0,            // 27 DrawWindow header
+    };
+
+    int m_dwMiscHeight[] = {
+        0x57342D,            // 0 StretchY
+        0x57473B,            // 1 Radio Icons
+        0x574783,            // 2 Radio Icons
+        0x57482F,            // 3 Radio Icons
+        0x574879,            // 4 Radio Icons
+        0x57659A,            // 5 2d Brief triangles
+        0x5765E2,            // 6 2d Brief triangles
+        0x576625,            // 7 2d Brief triangles
+        0x576668,            // 8 2d Brief triangles
+        0x576709,            // 9 2d Brief triangles
+        0x576751,            // 10 2d Brief triangles
+        0x576794,            // 11 2d Brief triangles
+        0x5767D7,            // 12 2d Brief triangles
+        0x574EAA,            // 13 Stats bars
+        0x574F0C,            // 14 Stats bars
+        0x719D47,            // 15 Font fixes
+        0x719D7C,            // 16 Font fixes
+        0x719DB5,            // 17 Font fixes
+        0x719DF2,            // 18 Font fixes
+        0x719E2F,            // 19 Font fixes
+        0x719EBF,            // 20 Font fixes
+        0x719EE3,            // 21 Font fixes
+        0x719C27,            // 22 Font fixes
+        0x719C58,            // 23 Font fixes
+        0x728864,            // 24 BarChart fixes
+        0x7288A9,            // 25 BarChart fixes
+        0x573F7D,            // 26 DrawWindow header
+        0x573FD6,            // 27 DrawWindow header
+    };
+
+    for (int i = 0; i < sizeof(m_dwMiscWidth) / sizeof(const void*); i++)
+    {
+        if (m_dwMiscWidth[i] != NULL)
```

Comment:
Why are there NULLs in the array that are checked here?

Reply:
removed